specs/tree250_su.json

```diff
@@ -5,7 +5,7 @@
     "prior": 1e-6,
     "env_size": [10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 110, 120, 130, 140, 150, 160, 170, 180, 190, 200, 210, 220, 230, 240, 250],
     "n_episodes": 5100,
-    "policy_arg": 1e-3,
+    "policy_arg": 1.0,
```

Comment:
Did you confirm this gets OK results?

Reply:
For Tree, it degrades the results, and seems 5.0 works as well as the original code.